### PR TITLE
moved store into app.js

### PR DESCRIPTION
**Pull request by jefflueck**
moved store code into app.js file

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 
 module.exports = {
-  entry: './src/store/configureStore.js',
+	entry: './src/app.js',
```
Comment (timgremore) [inline webpack.config.js]: @jefflueck this is nitpicking but feedback you will get from most PR reviews - `entry` should be aligned vertically with `output`

**Review by timgremore — CHANGES_REQUESTED** [review]
👍 this looks great! Nice work!

File: webpack.config.js (hunk 1 above)
Comment (jefflueck) [reply webpack.config.js]: Ok not sure what you mean by that. Are you saying the code order should be realigned, or that the output needs to change?

Comment (timgremore) [reply webpack.config.js]: yep - `entry` should be aligned with `output` (and `module`, `devtool`, `devServer`)

**Comment by timgremore** [comment]
Looks great @jefflueck - would you please squash these 2 commits into 1? After that we can merge. Here is one article explaining rebasing: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Pull request closed without merging**